int32 id
string waste_type
